CMakeLists.txt

```diff
@@ -71,7 +71,9 @@ find_package(OpenSSL 1.1.1 REQUIRED)
 set_target_properties(OpenSSL::SSL PROPERTIES
   INTERFACE_COMPILE_DEFINITIONS OPENSSL_NO_SSL2
 )
+set(SECP256K1_INSTALL TRUE)
 add_subdirectory(src/secp256k1)
+add_library(secp256k1::secp256k1 ALIAS secp256k1)
```

Comment:
Why were these two lines added?

Reply:
The previous `CMakeLists.txt` in `src/secp256k1` was written by us. The new `CMakeLists.txt` was written by the Bitcoin developers. The `SECP256K1_INSTALL` option is necessary to get it to include `install` instructions, and they don't define the `ALIAS` library (a CMake best practice) like we were.